packages/core/src/lib/wallet/types.ts

```diff
@@ -27,7 +27,7 @@ export type WalletConnect = {
   offlineSigner: OfflineSigner;
 };
 
-export type WalletWindowKey = 'keplr' | 'leap' | 'coin98' | 'falcon';
+export type WalletWindowKey = 'keplr' | 'leap' | 'coin98' | 'falcon' | 'fin';
```

Comment:
Just wanted to confirm, does Fin support the following methods in the `window.fin` object?

* enable
* getOfflineSignerAuto
* signArbitrary / verifyArbitrary

Reply:
Yes window.fin already support 
- enable
- getOfflineSignerAuto
- signArbitrary
Does not support
verifyArbitrary We recommend this function's handling by the developer.

Reply:
Great, that's fine!